systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterIT.java

```diff
@@ -46,6 +47,12 @@
 public class AbstractClusterIT {
 
     private static final Logger LOGGER = LogManager.getLogger(AbstractClusterIT.class);
+    protected static final String ZK_IMAGE = "STRIMZI_DEFAULT_ZOOKEEPER_IMAGE";
+    protected static final String KAFKA_IMAGE = "STRIMZI_DEFAULT_KAFKA_IMAGE";
+    protected static final String CONNECT_IMAGE = "STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE";
+    protected static final String S2I_IMAGE = "STRIMZI_DEFAULT_KAFKA_CONNECT_S2I_IMAGE";
+    protected static final String TO_IMAGE = "STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE";
+    protected static final String DEFAULT_INIT_KAFKA_IMAGE = "STRIMZI_DEFAULT_INIT_KAFKA_IMAGE";
```

Comment:
Why using the `DEFAULT_` prefix here which is not used for the others ? I'm ok with both solutions (adding `DEFAULT_` to the others or removing from this one) but having just one consistent declaration.